2-ui/1-document/05-basic-dom-node-properties/article.md

```diff
@@ -62,394 +62,394 @@ alert( document.body instanceof Node ); // true
-  <!-- Comment -->
+  Hola
+  <!-- Comentario -->
   <script>
     let text = document.body.firstChild;
 *!*
-    alert(text.data); // Hello
+    alert(text.data); // Hola
 */!*
 
     let comment = text.nextSibling;
 *!*
-    alert(comment.data); // Comment
+    alert(comment.data); // Comentario
 */!*
   </script>
 </body>
 ```
 
-For text nodes we can imagine a reason to read or modify them, but why comments?
+Para los nodos de texto podemos imaginar una razón para leerlos o modificarlos, pero ¿por qué comentarios?
 
-Sometimes developers embed information or template instructions into HTML in them, like this:
+A veces, los desarrolladores incorporan información o instrucciones de plantilla en HTML, así:
 
 ```html
 <!-- if isAdmin -->
-  <div>Welcome, Admin!</div>
+  <div>¡Bienvenido, administrador!</div>
 <!-- /if -->
 ```
 
-...Then JavaScript can read it from `data` property and process embedded instructions.
+...Entonces JavaScript puede leerlo desde la propiedad `data` y procesar las instrucciones integradas.
 
-## textContent: pure text
+## textContent: texto puro
 
-The `textContent` provides access to the *text* inside the element: only text, minus all `<tags>`.
+El `textContent` proporciona acceso al *texto* dentro del elemento: solo texto, menos todas las `<tags>`.
 
-For instance:
+Por ejemplo:
 
 ```html run
 <div id="news">
-  <h1>Headline!</h1>
-  <p>Martians attack people!</p>
+  <h1>¡Titular!</h1>
+  <p>¡Los marcianos atacan a la gente!</p>
 </div>
 
 <script>
-  // Headline! Martians attack people!
+  // ¡Titular! ¡Los marcianos atacan a la gente!
   alert(news.textContent);
 </script>
 ```
 
-As we can see, only text is returned, as if all `<tags>` were cut out, but the text in them remained.
+Como podemos ver, solo se devuelve texto, como si todas las `<etiquetas>` fueran recortadas, pero el texto en ellas permaneció.
 
-In practice, reading such text is rarely needed.
+En la práctica, rara vez se necesita leer este tipo de texto.
 
-**Writing to `textContent` is much more useful, because it allows to write text the "safe way".**
+**Escribir en `textContent` es mucho más útil, porque permite escribir texto de "forma segura".**
 
-Let's say we have an arbitrary string, for instance entered by a user, and want to show it.
+Digamos que tenemos un string arbitrario, por ejemplo, ingresado por un usuario, y queremos mostrarlo.
 
-- With `innerHTML` we'll have it inserted "as HTML", with all HTML tags.
-- With `textContent` we'll have it inserted "as text", all symbols are treated literally.
+- Con `innerHTML` lo tendremos insertado "como HTML", con todas las etiquetas HTML.
+- Con `textContent` lo tendremos insertado "como texto", todos los símbolos se tratan literalmente.
 
-Compare the two:
+Compara los dos:
 
 ```html run
 <div id="elem1"></div>
 <div id="elem2"></div>
 
 <script>
-  let name = prompt("What's your name?", "<b>Winnie-the-pooh!</b>");
+  let name = prompt("¿Cuál es tu nombre?", "<b>Winnie-el-pooh!</b>");
```

Comment:
buuuhhh
¡el nombre de Winnie-the-pooh no se traduce, hereje!
¿o si?

Ademas el original esta mal porque Pooh es parte del nombre,
deberias mandar PR al repo ingles, fatal bug
se llama 
Winnie-the-Pooh
o 
Winnie Pooh


jaja no me hagas caso estoy quemado